### DISCUSSION TITLE
How to show custom message during builds?

### DISCUSSION THREAD

**Discussion by sunknudsen** [vercel/next.js#44359] [Help]
Currently, when I run `npm run build` while app is running (using `npm run start`), “Invariant: failed to load static page” error is thrown and visitors see “Internal Server Error”.

I would like to show custom message during builds asking users to reload page every few minutes.

**Comment by icyJoseph** [comment]
I think this falls more into the category of CI/CD strategies doesn't it? You basically want to build your new server, without taking down the current server. The moment your run `npm run build` the contents of `.next` are changed. Meanwhile, `npm run start` is feeding of that folder. 

And that already makes the assumption that `npm run start` is not stopped to run `npm run build`. Who knows! it all depends on how you got it setup. 

I guess you need to somehow catch outgoing 500 errors, and replace them with your own page, but that's not always straight forward.  If you had a server engine like Apache, or others, you could do this though.
